### ISSUE TITLE
Implement websockets functionality

### ISSUE THREAD

**Issue by fabiansd** [acntech/RoboSchool#8]
implement functionality to handle the communication with the Game
 - Receive state vector and reward
 - Send action

**Comment by kristiansvalland** [comment]
Required action by frontend-server-team:
Must expose a socket.io endpoint which accepts an event of type `step` and data which has the format
`{ "action": <action>, "username": <username> }`
where 
`<action>` is one of 

- `UP`,
- `LEFT`,
- `RIGHT`,
- `DOWN`,
- `DROP_BOMB` and 
- `DO_NOTHING`.

After the action is received by the server, it should immediately compute the next state, reward, and game status (done) and any additional info, and return a dict of the form
`{
    "next_state": <state_dict>,
    "reward": float,
    "done": True|False,
    "info": <dict>
}`

**Comment by kristiansvalland** [comment]
Frontend-server must also accept websocket with `event_type=set_rewards`
and the data is a dict
`{
    "kill": 200,
    "die": -300,
    "action": -1,
    ...,
}`
